### PR TITLE
[SHOT-3987] Be able to run tk-vred in batch mode

### PR DESCRIPTION
**Pull request by barbara-darkshot**

### CODE REVIEW

**Review by rob-aitchison — DISMISSED** [review]
👍🏻

**Review by staceyoue — DISMISSED** [review]
LGTM!

**Review by rob-aitchison — APPROVED** [review]

**Review by staceyoue — APPROVED** [review]

File: python/tk_vred/menu_generation.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class VREDMenuGenerator(object):
     Menu generation functionality for VRED.
     """
 
-    ROOT_MENU_TEXT = u"S&hotGrid"
+    ROOT_MENU_TEXT = "S&hotGrid"
```
Comment (staceyoue) [inline python/tk_vred/menu_generation.py:23]: Just checking that we meant to remove the `u` prefix (which makes the menu string unicode) and still works in VRED?

Comment (rob-aitchison) [reply python/tk_vred/menu_generation.py:23]: @staceyoue The `black` pre-commit formatter took it out - good spotting!
I just launched a handful of versions back through 2020.3 of VRED and the menu appears so it seems to be okay.
Thanks!

**Pull request merged**